<launch>
  <arg name="COM1" value="/dev/ttyS0"/> 
  <arg name="COM2" value="/dev/ttyS1"/> 
  <arg name="COM3" value="/dev/ttyS2"/> 
  <arg name="COM4" value="/dev/ttyS3"/> 

  <arg name="COM11" value="/dev/ttyUSB0"/> 
  <arg name="COM12" value="/dev/ttyUSB1"/>
  <arg name="COM13" value="/dev/ttyUSB2"/>
  
  <arg name="BANDRATE_115200" value="115200"/> 
  <arg name="BANDRATE_9600" value="9600"/> 
  <arg name="BANDRATE_2400" value="2400"/> 
  <arg name="BANDRATE_38400" value="38400"/> 
  
  <arg name="DEV_DIR_ADD_LF" value="100"/> 
  <arg name="DEV_DIR_ADD_LB" value="102"/> 
  <arg name="DEV_DIR_ADD_RF" value="101"/> 
  <arg name="DEV_DIR_ADD_RB" value="103"/> 

  
     <!-- run qtcontroller -->
	

	<node pkg="MCtrlNode" name="MCtrlNode"  type="MCtrlNode" output="screen" args="$(arg COM2) $(arg BANDRATE_115200)" />
	
	
</launch>
